app/views/users/edit.html.erb

```diff
@@ -1,51 +1,51 @@
 <h2>Edit User Information</h2>
 
 <%= form_with model: @user do |form| %>
-  <div>
+<div>
```

Comment:
All these divs are within the form.  It makes more sense for them to be indented